homeassistant/components/edl21/__init__.py

```diff
@@ -1 +1,22 @@
 """The edl21 component."""
+import asyncio
+import logging
+
+from homeassistant import config_entries, core
+
+from .sensor import DOMAIN
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant, entry: config_entries.ConfigEntry
+) -> bool:
+    """Set up platform from a ConfigEntry."""
+    hass.data.setdefault(DOMAIN, {})
+    hass_data = dict(entry.data)
+    hass.data[DOMAIN] = hass_data
```

Comment:
There is no need to store the entry data, as the platform also gets the entry in its setup.

Reply:
done